tests/test_interpolation.py

```diff
@@ -0,0 +1,182 @@
+import unittest
+import numpy as np
+import pybullet as p
+from pybullet_industrial.toolpath import ToolPath
+from pybullet_industrial.interpolation import linear_interpolation, circular_interpolation
+from scipy.spatial.transform import Rotation as R, RotationSpline
+
+
+class TestInterpolation(unittest.TestCase):
+
+    def test_linear_interpolation(self):
+        # Test both cases for linear interpolation
+
+        # Case 1: Test with both start and end orientations
+        start_point = np.array([0.0, 0.0, 0.0])
+        end_point = np.array([10.0, 10.0, 10.0])
+        samples = 5
+        start_orientation = p.getQuaternionFromEuler(
+            [0, 0, 0])  # Neutral orientation
+        end_orientation = p.getQuaternionFromEuler(
+            [0, np.pi / 2, 0])  # 90 degrees rotation
+
+        result_with_both_orientations = linear_interpolation(
+            start_point, end_point, samples,
+            start_orientation, end_orientation)
+
+        # Expected positions for both orientations case
+        expected_positions = np.array([
+            [0.0, 2.5, 5.0, 7.5, 10.0],
+            [0.0, 2.5, 5.0, 7.5, 10.0],
+            [0.0, 2.5, 5.0, 7.5, 10.0]
+        ])
+
+        # Generate expected orientations using RotationSpline for SLERP
+        start_rot = R.from_quat(start_orientation)
+        end_rot = R.from_quat(end_orientation)
+        rotations = R.from_quat([start_rot.as_quat(), end_rot.as_quat()])
+        t_vals = np.linspace(0, 1, samples)
+        slerp_spline = RotationSpline([0, 1], rotations)
+        expected_orientations_both = slerp_spline(t_vals).as_quat()
```

Comment:
Is it possible to have preset expected orientations without using the RotationSpline function? (analog to expected_positions)